python/ucxx/examples/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# =================================================================================
+# SPDX-FileCopyrightText: Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: BSD 3-Clause License
+# =================================================================================
+
+set(cython_sources python_future_task_app.pyx)
+set(linked_libraries ucxx::ucxx ucxx::python Python3::Python)
+
+rapids_cython_create_modules(
+  CXX
+  SOURCE_FILES "${cython_sources}"
+  LINKED_LIBRARIES "${linked_libraries}" ASSOCIATED_TARGETS ucxx_examples
```

Comment:
I'm a bit confused, what is the ucxx_examples target? I see a custom `run_examples` target created in the cpp/examples/CMakeLists.txt, is that what you were trying to refer to? Does this Python application need to link to something created in that example?

Reply:
No, this refers to https://github.com/rapidsai/ucxx/pull/46/files/7e0ad1b7de36915eb3edf78898e1208e5eebcca6#diff-691a68ea554b5fd97545a2986bb6b915ecb28e20ea05ec5cb95b8b94340a57be, see the [`cython_sources`](https://github.com/rapidsai/ucxx/pull/46/files/7e0ad1b7de36915eb3edf78898e1208e5eebcca6#diff-0cdf2e3fa52349932e0e3d2f84da5e2c9d3c664140dbe62c74db204c97ef4807R6).

This is a Cython file that [wraps some C++ code](https://github.com/rapidsai/ucxx/pull/46/files/7e0ad1b7de36915eb3edf78898e1208e5eebcca6#diff-774dabed7ea310b8dfa9acde01482c4ce18a255ee4229f59e7c3b47d9a342446) which demonstrates how to set the result/exception of a Python future from a C++ future.

Reply:
To clarify, I'm just confused about the `ASSOCIATED_TARGETS` argument here. I think that can be removed just like you removed the `rapids_cython_add_rpath_entries` above. However, if the example requires ucxx you may need to have ucxx rather than ucxx_examples here.

Reply:
Got it, it seems you're right and we don't need it. Removed it in https://github.com/rapidsai/ucxx/pull/46/commits/e55efdd7716786d8584e0cebd1ede9ca59091481 .

Reply:
Sorry, linked the wrong commit above, remove it in https://github.com/rapidsai/ucxx/pull/46/commits/8439f9527c38cd77d7ed425b5419b12a08de728d .